en/lessons/basics/strings.md

```diff
@@ -37,6 +37,14 @@ iex(6)> "hełło" <> <<0>>
 
 `322` is the Unicode codepoint for ł but it is encoded in UTF-8 as the two bytes `197`, `130`.
 
+You can get a character’s code point by using ?
+
+```elixir
+iex(8)> ?Z  
+90
+```
+This allows you to use the compact notation ?Z rather than 'Z'.
```

Comment:
I also dont understand this sentence a ton, but grok the premise of this as a whole. It doesn't really 'compact' it I dont think, but its notation does make it clear that its 'not a string' vs the single quote for learners as that is often a new foreign concept.